python/FactorySystem/Parser.py

```diff
@@ -132,6 +140,12 @@ def _parseNode(self, filename, node):
       # Put it in the warehouse
       self.warehouse.addObject(moose_object)
 
+    # Are we in a tree node that "looks" like it should contain a buildable object?
+    elif len(node.params.keys()):
+      print 'Error detected when parsing file "' + os.path.join(os.getcwd(), filename) + '"'
+      print '       Missing "type" parameter in block'
+      error_code = error_code | 0x20
```

Comment:
Yeah... I'm just adding on to the current mess. This can sit for awhile, it's actually not quite correct anyway. I forgot about the `active` parameter which is valid in a block that's not intended to be an object block